### ISSUE TITLE
Problems with compiling examples

### ISSUE THREAD

**Issue by szagi3891** [tokio-rs/tokio-minihttp#1]
```
grzegorz@grzegorz:~/Pulpit/rust/tokio-minihttp$ cargo run --example hello_world
    Updating git repository `https://github.com/tokio-rs/tokio-ssl`
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/tokio-rs/tokio`
    Updating git repository `https://github.com/carllerche/bytes`
error: no matching package named `tokio-ssl` found (required by `tokio-minihttp`)
location searched: https://github.com/tokio-rs/tokio-ssl
version required: *
grzegorz@grzegorz:~/Pulpit/rust/tokio-minihttp$ 
grzegorz@grzegorz:~/Pulpit/rust/tokio-minihttp$ 
grzegorz@grzegorz:~/Pulpit/rust/tokio-minihttp$ cargo run --example ssl_server
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/carllerche/bytes`
    Updating git repository `https://github.com/tokio-rs/tokio`
    Updating git repository `https://github.com/tokio-rs/tokio-ssl`
error: no matching package named `tokio` found (required by `tokio-minihttp`)
location searched: https://github.com/tokio-rs/tokio
version required: *
grzegorz@grzegorz:~/Pulpit/rust/tokio-minihttp$ 
grzegorz@grzegorz:~/Pulpit/rust/tokio-minihttp$ 
grzegorz@grzegorz:~/Pulpit/rust/tokio-minihttp$ cargo --version
cargo 0.13.0-nightly (c7ffb3e 2016-08-28)
```

**Comment by alexander-irbis** [comment]
It seems that tokio is under heavy refactoring and tokio-\* components structure is changed. We have to wait while minihttp will be updated in accordance with the idea.

**Comment by carllerche** [comment]
Gah yes! I will try to work on this tomorrow.

**Comment by szagi3891** [comment]
Thank you.

**Comment by chrivers** [comment]
I can confirm that the example compiles now. However, it still gets a timeout with ab (apache benchmark), #2

**Comment by carllerche** [comment]
Sorry for the churn & delay. This should be fixed: 7d8c4491ae720ce3ee1002e1cde56cd7e684329c

**Comment by szagi3891** [comment]
Cool. Thank you :)
